bin/fin

```diff
@@ -3097,13 +3105,13 @@ config_generate ()
 	if [[ -f ${yml_file} ]]; then
 		if [[ -d .docksal ]]; then
 			echo-yellow ".docksal directory already exists."
-			_confirm "Remove it and start from scratch?"
-			rm -rf ./.docksal
+			_confirm "Overwrite?"
```

Comment:
I think having 2 commands is too much. The only function of this command is to copy a proper file from one dir to another. Instead of remembering this command users can just use their file manager to copy. If there's an additional command to remember which copies without deleting, and another one that copies with deleting I feel that efforts of remembering them will not cover for benefits and it will never be used.